sector-builder/src/scheduler.rs

```diff
@@ -300,7 +258,7 @@ impl<T: KeyValueStore, S: SectorStore> SectorMetadataManager<T, S> {
         piece_path: String,
         store_until: SecondsSinceEpoch,
     ) -> Result<SectorId> {
-        let destination_sector_id = add_piece(
+        let destination_sector_id = crate::helpers::add_piece(
```

Comment:
"good rust rule": import the top level, e.g. in this case

```rust
use crate::helpers;

helpers::add_piece()
```

Reply:
done!